bootstrap

```diff
@@ -18,7 +18,7 @@ if [ -f /etc/debian_version ]; then
 else 
     packages=(which libevent-devel libffi-devel libxml2-devel libxslt-devel zlib-devel)
     if [ -f /etc/fedora-release ]; then
-        packages+=(python2-pip python2-virtualenv python2-devel)
+        packages+=(python-virtualenv python3-devel libevent-devel libffi-devel libxml2-devel libxslt-devel zlib-devel)
```

Comment:
the libevent-devel, libffi-devel, libxml2-devel, libxslt-devel and  zlib-devel  are already in the packages array here: https://github.com/ceph/s3-tests/pull/337/files#diff-ca4c50b905dc21ea17a10549a6f5944fR19